### PR TITLE
Many updates: refs, just one author, clean up and copy-editing.

### PR DESCRIPTION
**Pull request by WarrenWeckesser**

### CODE REVIEW

File: papers/scipy/scipy.rst [hunk 1]
```diff
@@ -26,36 +18,56 @@
...
+interpolation, integration, linear algebra, Fourier transforms, signal
+and image processing, ODE solvers, special functions, sparse matrices,
+and more.
+
+The ``signal`` subpackage within the SciPy library includes tools
+for several areas of computation, including signal processing, interpolation,
+linear systems analysis and even some elementary image processing.  In this
+Cookbook chapter, we focus on specific subset of the capabilities of
+of this subpackage: the design and analysis of linear filters for discrete-time
+signals.
```
Comment (rgommers) [inline papers/scipy/scipy.rst]: Maybe add a sentence here like: "These filters are commonly used in digital signal processing (DSP) for audio signals, as well as in <control systems, etc.> ..."

File: papers/scipy/scipy.rst [hunk 2]
```diff
@@ -26,36 +18,56 @@
     same size as and side-by-side with the z.  Using `a_{_N}` makes it
     very clear that N is a subscript of a.
 
------
-SciPy
------
+-------------------------------------
+Discrete-time Linear Filters in SciPy
```
Comment (rgommers) [inline papers/scipy/scipy.rst]: As a title this sounds a bit technical, maybe something like "Signal processing with SciPy - linear filters"?

**Comment by rgommers** [comment]
LGTM besides the two comments I made

File: papers/scipy/scipy.rst (hunk 1 above)
Comment (WarrenWeckesser) [reply papers/scipy/scipy.rst]: Good idea.  I went with "These filters are commonly used in digital signal processing (DSP) for audio signals and other time series data with a fixed sample rate."

File: papers/scipy/scipy.rst (hunk 2 above)
Comment (WarrenWeckesser) [reply papers/scipy/scipy.rst]: Agreed, "signal processing" should be in the title.  I switch the hyphen to a colon: "Signal processing with SciPy: linear filters".

**Comment by WarrenWeckesser** [comment]
Thanks Ralf!  I just pushed an updated version.

**Pull request merged**